src/OmnichannelChatSDK.ts

```diff
@@ -1546,7 +1548,7 @@ class OmnichannelChatSDK {
                 const surveyInviteLinkRequest = {
                     "FormId": msfp_sourcesurveyidentifier,
                     "ConversationId": conversationId,
-                    "OCLocaleCode": getLocaleStringFromId(localeId)
+                    "OCLocaleCode": getLocaleStringFromId(this.localeId)
```

Comment:
do we have a validation in place for locale format?, in this case, if the string doesn't have a proper format , we can default immediately to 'en-us',  instead of doing a search for the string in the array for an invalid format, at least in terms of length we could do a validation

Reply:
`getLocaleStringFromId()` would attempt to find the locale string based on the locale id. If the locale id is invalid, then it would return an `""`. In that case, it would fallback to "en-us".